frontend/src/css/components/ExpenseItem.css

```diff
@@ -0,0 +1,59 @@
+.ExpenseItem {
+  border-color: rgba(48, 50, 51, .2);
+  border-radius: 6px;
+  box-shadow: 0 1px 3px rgba(0,0,0,.06);
+}
+
+.ExpenseItem-category {
+  min-width: 100px;
+}
+
+.ExpenseItem-category-icon-wrapper {
+  width: 3.5rem;
+  height: 3.5rem;
+  background-color: rgba(0,0,0,0.1);
+}
+
+.ExpenseStatus {
+  font-size: 9px;
+  padding: 2px 6px;
+  border-radius: 4px;
+}
+
+.category-default {
```

Comment:
I wasn't thinking about BEM notation here, I just wanted a very specific name that will not affect other elements on the page. Do you have any suggestion for naming convention?